src/modules/account/submodule/multisig/views/ViewMultisigHome.vue

```diff
@@ -185,7 +184,6 @@ export default {
         isCosigner,
         multisigAccountsList,
         cosignerAccountsList,
-        explorerLink
```

Comment:
there's a explorerLink variable used in line 37

Reply:
noted, will remove it